### DISCUSSION TITLE
Getriebe optionen

### DISCUSSION THREAD

**Discussion by Basti06** [Mogli12/KeyboardSteer#834] [Ideas]
Wollte mal fragen ob es wie im 19er ne Option geben wird das man sich das passende Getriebe raussuchen kann?
weil was mir aufgefallen ist das mein beim Varion z.b. keine 2 Fahrstufen hat oder aber ich habe dieses übersehen

**Comment by Mogli12** [comment]
Da FS22 bereits ein manuelles Getriebe hat, gibt es keine Pläne, dass VCA das Getriebe überschreibt.
